Corgibytes.Freshli.Cli/Functionality/Git.cs

```diff
@@ -100,69 +100,66 @@ private void Delete()
 
     private void Clone(string gitPath)
     {
-        var cloneProcess = new Process
-        {
-            StartInfo = new ProcessStartInfo()
-            {
-                FileName = gitPath,
-                WorkingDirectory = Directory.FullName,
-                Arguments = $"clone {Url} .",  // clone directly in the working directory
-                RedirectStandardOutput = true,
-                RedirectStandardError = true
-            }
-        };
-        cloneProcess.Start();
-        cloneProcess.WaitForExit();
-
-        if (cloneProcess.ExitCode != 0)
+        var stdErrBuffer = new StringBuilder();
+        var command = CliWrap.Cli.Wrap(gitPath).WithArguments(
+                args => args
+                    .Add("clone")
+                    .Add(Url)
+                    .Add('.')
+            )
+            .WithValidation(CommandResultValidation.None)
+            .WithWorkingDirectory(Directory.FullName)
+            .WithStandardErrorPipe(PipeTarget.ToStringBuilder(stdErrBuffer));
```

Comment:
We need to add `using` here to make sure that the `CommandTask<T>` instance is properly cleaned up, because it implements `IDisposable`.

```suggestion
        using var command = CliWrap.Cli.Wrap(gitPath).WithArguments(
                args => args
                    .Add("clone")
                    .Add(Url)
                    .Add('.')
            )
            .WithValidation(CommandResultValidation.None)
            .WithWorkingDirectory(Directory.FullName)
            .WithStandardErrorPipe(PipeTarget.ToStringBuilder(stdErrBuffer));
```

Reply:
Does not work. If you make this change in Jetbrains, it will immediately alert you with the following:

```
Type 'CliWrap.Command' used in a 'using' declaration must be implicitly convertible to 'System.IDisposable'|
```

Reply:
I liked the solution that you came up with. Thanks!